src/imitation/algorithms/bc.py

```diff
@@ -334,18 +334,19 @@ def __init__(
         self._bc_logger = BCLogger(self.logger)
 
         self.action_space = action_space
-        self.observation_space = observation_space
+        obs_space_without_rgb = wrappers.remove_rgb_obs_space(observation_space)
```

Comment:
Why is this the responsibility of the algorithm? This will mean we have to modify every algorithm where interactive data collection is supported. More generally the algorithm is going to have to guess what the desired behavior is from the observation space.